tests/login/test_web.py

```diff
@@ -26,11 +28,11 @@ def create_client(service: Mock) -> Any:
     return app.test_client()
 
 
-def get_token(role: str = Role.USER) -> Dict[str, str]:
+def get_access_token(role: str = Role.USER) -> Dict[str, str]:
     if role == Role.USER:
-        token = "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpYXQiOjE2MTIxNzU4MDYsIm5iZiI6MTYxMjE3NTgwNiwianRpIjoiMzkzM2M0MTItZGI2NS00YjUwLTk5ZGMtYzJlYjgxMTBkNTlhIiwiZXhwIjo5OTk5OTk5OTk5LCJpZGVudGl0eSI6eyJpZCI6MCwibmFtZSI6ImFkbWluIiwicm9sZSI6IlVTRVIifSwiZnJlc2giOmZhbHNlLCJ0eXBlIjoiYWNjZXNzIn0.uNpLBLA-tEM-TB8dv4wrj3KLGVQL9A07wjE3835GDjM"
+        token = "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJmcmVzaCI6ZmFsc2UsImlhdCI6MTYxMzQ2NzA3NSwianRpIjoiMTZlZDc3NmMtNmNlMS00MDdhLWFkMmItMzFiN2M0NjkyOTA5IiwibmJmIjoxNjEzNDY3MDc1LCJ0eXBlIjoiYWNjZXNzIiwic3ViIjp7ImlkIjoxLCJuYW1lIjoidXNlciIsInJvbGUiOiJVU0VSIn0sImNzcmYiOiI0ODIzYTlmYi1mNzJiLTQxZTAtODk1Mi1lYWU3MjlkNGUxZjQiLCJleHAiOjk5OTk5OTk5OTl9.QIcMKIEm_rE_0KrNgWMKE9ykAU9eHcbGckGFE8hhTk0"
     elif role == Role.ADMIN:
-        token = "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpYXQiOjE2MTIxNzU4MDYsIm5iZiI6MTYxMjE3NTgwNiwianRpIjoiMzkzM2M0MTItZGI2NS00YjUwLTk5ZGMtYzJlYjgxMTBkNTlhIiwiZXhwIjo5OTk5OTk5OTk5LCJpZGVudGl0eSI6eyJpZCI6MCwibmFtZSI6ImFkbWluIiwicm9sZSI6IkFETUlOIn0sImZyZXNoIjpmYWxzZSwidHlwZSI6ImFjY2VzcyJ9.Gbxhmz6XSke0OZ2-fwRFKoh8LOrFj0pbJeR3Vfj536Q"
+        token = "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJmcmVzaCI6ZmFsc2UsImlhdCI6MTYxMzQ2NzA3NSwianRpIjoiMTZlZDc3NmMtNmNlMS00MDdhLWFkMmItMzFiN2M0NjkyOTA5IiwibmJmIjoxNjEzNDY3MDc1LCJ0eXBlIjoiYWNjZXNzIiwic3ViIjp7ImlkIjowLCJuYW1lIjoiYWRtaW4iLCJyb2xlIjoiQURNSU4ifSwiY3NyZiI6IjQ4MjNhOWZiLWY3MmItNDFlMC04OTUyLWVhZTcyOWQ0ZTFmNCIsImV4cCI6OTk5OTk5OTk5OX0.hvvGLnqbRzbuGyCmjuecX5-puMYDDOwsFXZHiUMJyFk"
```

Comment:
You should generate the access_token. Also (for refresh test), it should be generated to be expired

Reply:
Flask-JWT generates token only inside a request. Therefore we have choice between:
- hardcoded token like these with very long expiration delay (2 centuries...)
- request endpoint /login to generate token before each other endpoint tests. This solution avoid hardcoded token, but this is not simple unit test, since all endpoint need /login endpoint to success

Test expiration session with refresh token should be done, but inside integration or E2E test.

Reply:
Why can't you use create_access_token with a negative timedelta (for the expired token) ?

Reply:
no it returns 401 unauthorize...., so i set expiration date at 99999999